ql/src/semmle/go/controlflow/ControlFlowGraphImpl.qll

```
@@ -918,7 +918,7 @@ module CFG {
     }
 
     override Completion getCompletion() {
-      not getTarget().mustPanic() and
+      (not exists(getTarget()) or getTarget().mayReturnNormally()) and
```

As discussed previously, we cannot silently drop support for `mustPanic`. We need to deprecate it and continue supporting it for a while.